SECURITY.md

```diff
@@ -96,15 +96,27 @@ Here is a list of steps you should follow to increase the security of your insta
 
 ### Prevent external access
 
-Exposing the eLabFTW service only internally is a good way to avoid having the whole world attacking it.
+Only exposing the eLabFTW service is a good way to avoid having the whole world attacking it.
 
 ### Add a reverse proxy
 
-A reverse proxy in front of the eLabFTW container can allow you to enable additional mitigation strategies such as Apache's modsecurity or a Web Application Firewall.
+A reverse proxy in front of an eLabFTW container can allow you to enable additional mitigation strategies such as Apache's modsecurity or a Web Application Firewall. By using eLabFTW's external authentication mechanism in conjunction with eg `mod_auth_mellon`, the risk of unauthenticated users even reaching the system can be mitigated further (while allowing authentication information to propagate).
 
 ### Enable 2FA
 
-You can enforce multi factors authentication (with TOTP mechanism) for Sysadmins, Admins or all users. It is also important to educate users on the importance of password managers and the use of unique and long passwords.
+You can enforce multi factor authentication (with TOTP mechanism) for Sysadmins, Admins or all users. It is also important to educate users on the importance of password managers and the use of unique and long passwords.
+
+### Using single sign-on? Turn off local user (password) registration and login
+
+If eLabFTW is set up to use some external source of identity information (such as SAML auth or external auth), you can turn off the ability to log in- or register using local accounts. This can be re-enabled by an administrator with direct database access, in case the IDP setup fails.
+
+### Turn on optional features only if required
+
+There are multiple optional features in eLabFTW. In a conservative setup, do consider not turning them on (eg to not allowing link-sharing).
```

Comment:
same eg remark

Reply:
english is also a bit weird. Try `There are multiple optional features in eLabFTW. In a conservative setup, consider keeping them disabled (e.g., to prevent link-sharing).`

Reply:
>  In a conservative setup, do consider keeping them disabled (e.g. prevent link-sharing unless you have investigated the ramifications of allowing it).

https://github.com/elabftw/elabftw/pull/5355/commits/55e555dec0a9e3fb2eaad58d400a69c7e7575c6c